App.js

```diff
@@ -1,26 +1,326 @@
+      setQuestion('two')
+      afterAnswering()
+    } else if(question==='two') {
+      setQuestion('three')
+      afterAnswering()
+    } else if(question==='three') {
+      setQuestion('four')
+      afterAnswering()
+    } else if(question==='four') {
+      setQuestion('five')
+      afterAnswering()
+    } else if(question==='five') {
+      setQuestion('summary')
+      afterAnswering()
+    }
+  }
+
+  const rightAnswer = () => {
+    setScore(score+1)
+    setHide(true)
+    setRight(true)
+  }
+
+  const wrongAnswer = () => {
+    setHide(true)
+    setWrong(true)
+    return Vibration.vibrate(duration)
+  }
+
+  const restartQuiz = () => {
+    setScore(0)
+    setQuestion('begin')
+  }
+
+
   return (
     <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
+      {question === 'begin' && (
+      <View>
+        <Container>
+        <Title>Welcome to Michels movie quiz</Title>
+        <Title>🍿🍿🍿</Title>
+        <Title>❓❓❓</Title>
+        <Button>
+          <ButtonText onPress={() => setQuestion('one')}>Start the quiz</ButtonText>
+        </Button>
+        <LottieView source={require('./1879-movie-loading.json')} autoPlay loop style={{height: 100}}/>
+        </Container>
+      </View>
+    )}
+     
+    {question === 'one' && (
+      <View>
+        <Container>
+        <Title>Question 1</Title>
+        <Title>How many movies with Leonardo DiCaprio have Martin Scorsese directed?</Title>
+        {!hide && (
+        <View>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Three</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={rightAnswer}>Five</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Four</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Two</ButtonText>
+          </Button>
+        </View>
+        )}
+        {right && (
+          <TitleRight>Correct answer! Please go to the next question</TitleRight>
+        )}
+        {wrong && (
+          <TitleWrong>Wrong answer! Please go to the next question</TitleWrong>
+        )}
+        <Score>{score} points</Score>
+        </Container>
+      </View>
+    )}
+
+    {question === 'two' && (
+      <View>
+        <Container>
+        <Title>Question 2</Title>
+        <Title>Which one of these actors has won an Oscar for best actor?</Title>
+        {!hide && (
+        <View>
+          <Button>
+            <ButtonText onPress={rightAnswer}>Jamie Foxx</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Johnny Depp</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Tom Cruise</ButtonText>
+          </Button>
+          <Button>
+            <ButtonText onPress={wrongAnswer}>Daniel Craig</ButtonText>
+          </Button>
```

Comment:
This could be extracted further into the "options component" and here you could use checkboxes or radio buttons then do a function to check if the selected one is the correct one. I don't know if you want me to explain about this further, or if this is enough? (based on your Slack comment)